content/pages/what-is-mongock.md

```diff
@@ -7,7 +7,49 @@ eleventyNavigation:
-**NEEDS TO BE CHANGED**
+Mongock is a Java based Migration tool as part of your Application code. It allows Developers to have ownership and control over Data migrations during the deployment process, allowing safer migrations in Distributed Systems as code and data changes are shipped together. 
 
+It was originally designed for MongoDB Data migrations and the product is evolving to provide a wider suite of NoSQL / SQL Database compatibility and cover other feautures for transactional execution use cases that require state management in Distributed systems. 
 
+**Why Mongock?**
+Our mission is to enable Developers to deploy and manage stateful Data migrations in Distributed Systems in a safe and reliable manner. If you are looking for migrating data for your Java Application Services in a safe and reliable manner via Code, Mongock is your best bet.
+
+There are several features Mongock enables you in your project:
+- Mongock is a Java-based tool that you can import in your Application.
+- Mongock promotes a Code-first approach for Migrations, allowing you to write your migration scripts in Java wh will ship with your Application code.
+- Mongock persists in your DB the changes made, allowing you to gain more control over the management of DB changes via Migrations, Rollbacks + other amazing features.
+- It is the most reliable production-grade solution for MongoDB migrations currently in the market, compatible with Mongo Atlas and different MongoDB versions.
+- Distributed solution with solid locking mechanism.
+- Our team offers great support and is very responsive.
+- We are an open source tool, operating under the Apache License 2.0 
+- We maintain and update features regularly.
+- Currently  Used by several tech companies in different industries.
+- Adopted by well known frameworks such as JHipster as part of the scaffolding.
+- Can be used together with most, if not all, frameworks.
+- Can run in Standalone projects without depending on frameworks.
+- Great support with the Spring framework overall, with native implementations in SpringBoot.
+
+Get more information about our support model at dev@cloudyrock.io​ and we can help you walking you to production. 
+
+​
+**History**
+Mongock began it's journey as a fork from the unmantained MongoBee repository, providing a more reliable locking mechanism during migrations. We considered this enhancement to be critical given the importance of this core feature in Distributed Systems to provide a production-ready solution. 
+
+We have then invested our time in supporting it's adoption and enhancing features since. We are now shaping the Mongock product to expand further its capabilities and features, enabling us to Migrate safely across any Database or REST APIs using our distributed locking mechanism.
+
+If you'd like to help us during this journey, please don't hesitate to contact us _here_!
```

Comment:
_here_ should be dev@cloudyrock.io, although we should take advantage of change to open the dev@mongock.io email and maintin  both for a while , unti lwe feel we are in a good place to remove dev@clourydock.io

Reply:
We can put both for the time being and create a Forwarding  rule between inboxes. We would want to move to the right one eventually, it's more about deciding when the correct time is. I was thinking this release given the changes and upcoming backlog.

Reply:
Nice. I agree in everythig, except "puting both", I would keep both active, but only put the new one to start the "migration".

But we'll do this in a separte ticket